packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -286,14 +295,10 @@ function ViewGrid< Item >( {
 		{ regularFields: [], badgeFields: [] }
 	);
 	const hasData = !! data?.length;
-	const updatedPreviewSize = useUpdatedPreviewSizeOnViewportChange();
 	const hasBulkActions = useSomeItemHasAPossibleBulkAction( actions, data );
-	const usedPreviewSize = updatedPreviewSize || view.layout?.previewSize;
-	const gridStyle = usedPreviewSize
-		? {
-				gridTemplateColumns: `repeat(${ usedPreviewSize }, minmax(0, 1fr))`,
-		  }
-		: {};
+	const usedPreviewSize = view.layout?.previewSize;
+	// This is the maximum width that an image can achieve in the grid.
+	const size = '900px';
```

Comment:
As a gift to our future selves, would you expand why this is the maximum width an image can achieve in the grid?

Reply:
Sure!